chatcommands.py

```diff
@@ -1189,7 +1189,7 @@ def delete_force(msg):
     """
     # noinspection PyBroadException
     try:
-        msg.delete()
+        msg._client._do_action_despite_throttling(("delete", msg.id, ""))
```

Comment:
Could you make a helper for this (i.e. `messages.delete(message)`)?

Reply:
Sure, I'll include the try too